web/unit_tests/test.sh

```diff
@@ -118,9 +118,10 @@ fi
 BASE_PATH=`dirname $BASH_SOURCE`
 cd $BASE_PATH/../source
 
-./build_dev_resources.sh
+# Compile our testing dependencies; make sure the script fails if compilation fails!
```

Comment:
Maybe add bash flags like in `oem/firstvoices/common/build_keyboards.sh` (ll. 12-18)

```
# Set sensible script defaults:
# set -e: Terminate script if a command returns an error
set -e
# set -u: Terminate script if an unset variable is used
set -u
# set -x: Debugging use, print each statement
# set -x

## START STANDARD BUILD SCRIPT INCLUDE
# adjust relative paths as necessary
THIS_SCRIPT="$(greadlink -f "${BASH_SOURCE[0]}" 2>/dev/null || readlink -f "${BASH_SOURCE[0]}")"
. "$(dirname "$THIS_SCRIPT")/../../../resources/build/build-utils.sh"
## END STANDARD BUILD SCRIPT INCLUDE
```